app/lib/nodemailer.server.ts

```diff
@@ -0,0 +1,21 @@
+import { createTransport } from 'nodemailer';
```

Comment:
I wonder if we should rename this to `app/lib/notifications.server.ts` and export a `send()` method only.  This way we can keep the details about nodemailer hidden from the rest of the code.

Basically, we'd take what you're doing here and add:

```ts
// https://nodemailer.com/usage/
export default function send(data) {
  const transport = initializeTransport();
  logger.debug('Sending notification', data);
  return transport.send(data);
}
```

Reply:
Agreed. We shouldn't expose `initializeTransport.`.